jen-jin/moment
# Login Screen

Targets: [Login Ticket](https://speaku.atlassian.net/browse/FYDP-15)

Details
- Created the UI for the login screen with username and password
- Commented out logic for server validation
- Added default styles from figma
- No tests written yet

Testing
- npm run/yarn start
- Navigate to /login for your localhost
- Confirm login screen shows with username and password
- When you press login, you should be navigated to the dashboard page
- Typing in the password is covered by dots (for privacy)
